volatility3/framework/plugins/windows/hollowprocesses.py

```diff
@@ -0,0 +1,188 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+import logging
+from typing import NamedTuple
+
+from volatility3.framework import interfaces, exceptions, constants
+from volatility3.framework import renderers
+from volatility3.framework.configuration import requirements
+from volatility3.framework.objects import utility
+from volatility3.plugins.windows import pslist, vadinfo
+
+vollog = logging.getLogger(__name__)
+
+VadInfo = NamedTuple(
```

Comment:
I think `namedtuple` is canonically defined in `collections` not in `typing`?  I think that's just the type of it...
https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
I can change it if needed, but I got the flow for it from here: https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/interfaces/renderers.py#L21-L29

Reply:
Fair enough, I don't think that's right either and I'd prefer we went with collections, but it is used elsewhere so I won't quibble.  Maybe one day I'll have a run back over the codebase and change them both...  5:)